### ISSUE TITLE
Cookie value can be boolean?

### ISSUE THREAD

**Issue by David-Rickard** [andreizanik/cookies-next#33]
```ts
export declare const getCookie: (key: string, options?: OptionsType) => CookieValueTypes;
export declare type CookieValueTypes = string | boolean | undefined | null;
```

Why can `getCookie()` return a boolean? A cookie is either a `string` or it doesn't exist, right? Is there some extra layer of parsing this library is doing? I would expect the return value to be `string | undefined`.

**Comment by jleclanche** [comment]
I had this exact same "what" moment. It does look like the library is doing extra parsing:

https://github.com/andreizanik/cookies-next/blob/d87cd88b8820a3421b34eb21965ccd6cb4fcbebd/src/index.ts#L7-L13

IMO it's super out of place. I kind of understand why it's trying to do this (reverse input for when giving cookies a non-string input), but @andreizanik please consider the case where eg. you're storing someone's username in a cookie and you run into obscure bugs when users try to pick the username `true` or `false`, or some such. A string is a string is a string. Additionally, going down the parsing path, you have the question of numbers etc.

@andreizanik If you agree, I will happily file a PR removing the parsing.

**Comment by andreizanik** [comment]
@jleclanche 
I agree, but this update will change the package version to 3.0.0
If the version is not raised to 3.0.0, the upgrade may break current projects that rely on this additional parsing.

Thanks!

**Comment by alanhamlett** [comment]
Any updates on this? Would love to only have strings returned from my cookies.

**Comment by andreizanik** [comment]
I fixed this problem in version 3.0.0

https://github.com/andreizanik/cookies-next/releases/tag/v3.0.0
